vars/podman.groovy

```diff
@@ -0,0 +1,85 @@
+//Run podman remote build and push
+// Available parameters:
+//    arch:              string -- Build architecture
+//    tag:               string -- Image tag
+//    repo:              string -- Quay repository 
+def buildRemote(params = [:]) {
+    arch = params['arch']
+    repo = params['repo']
+    tag = params['tag'] 
+
+    withCredentials([
+        string(credentialsId: "fcos-${arch}-builder-host-string",
+               variable: 'REMOTEHOST'),
+        string(credentialsId: "fcos-${arch}-builder-uid-string",
+               variable: 'REMOTEUID'),
+        sshUserPrivateKey(credentialsId: "fcos-${arch}-builder-sshkey-key",
+                          usernameVariable: 'REMOTEUSER',
+                          keyFileVariable: 'CONTAINER_SSHKEY')
+    ]) {
+        withEnv(["CONTAINER_HOST=ssh://${REMOTEUSER}@${REMOTEHOST}/run/user/${REMOTEUID}/podman/podman.sock"]) {
+            shwrap("""
+            # workaround bug: https://github.com/jenkinsci/configuration-as-code-plugin/issues/1646
+            sed -i s/^----BEGIN/-----BEGIN/ \$CONTAINER_SSHKEY
+            """)
+            shwrap("""
+               podman --remote build --tag $repo:$tag -f src/Dockerfile
+               podman --remote push $repo:$tag > /tmp/tags.txt
+            """)
+            cleanImages()
+        }
+    }
+}
+// Create and push a manifest list
+// Available parameters:
+//    tag:               string -- Image tag
+//    repo:              string -- Quay repository
+def createManifest(params = [:]) {
+    repo = params['repo']
+    tag = params['tag'] 
+
+    shwrap("""
+        podman manifest create $repo:main
+        podman manifest add --features cosa_commit=$tag $repo:main docker://$repo:$tag
+        podman manifest push --all $repo:main docker://$repo:main  --remove-signatures -f v2s2
+    """)
+}
+// Remove all local images and containers:
+def cleanImages(params = [:]) {
+    shwrap("""
+        podman rmi -a -f
+    """)
+} 
+// Run podman remote build and push
+// Available parameters:
+//    arch:              string -- Build architecture
+//    dir:               string -- Directory for source
+//    repo:              string -- Quay repository
```

Comment:
This looks like it's used both as the git repo and the registry repo. Was this supposed to be two separate parameters instead?